tests/src/ExampleAttributesCommandFile.php

```diff
@@ -26,18 +27,12 @@ public function setOutput($output)
         $this->output = $output;
     }
 
-    #[CommandLineAttributes(
-        name: 'my:echo',
-        description: 'This is the my:echo command',
-        help: "This command will concatenate two parameters. If the --flip flag\nis provided, then the result is the concatenation of two and one.",
-        aliases: ['c'],
-        usage: ['bet alpha --flip' => 'Concatenate "alpha" and "bet".'],
-        options: [
-            'flip' => [
-                'description' => 'Whether or not the second parameter should come first in the result. Default: false'
-            ]
-        ]
-    )]
+    #[CLI\Command(name: 'my:echo', aliases: ['c'])]
+    #[CLI\Help(description: 'This is the my:echo command', synopsis: "This command will concatenate two parameters. If the --flip flag\nis provided, then the result is the concatenation of two and one.",)]
+    #[CLI\Param(name: 'one', description: 'The first parameter')]
```

Comment:
Maybe this should be CLI\Arg or CLI\Argument? The docblock code allowed @param or @arg, but usually the former was used, because that's the PHP standard for documenting a function parameter. With attributes, we don't need to overload the docblock tags, so we can name things after the thing they actually are, which in this case is a commandline argument.

Reply:
Done with `CLI\Argument`?